cli/commands/project.go

```diff
@@ -0,0 +1,127 @@
+package commands
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+
+	"github.com/phase2/rig/cli/util"
+	"github.com/urfave/cli"
+)
+
+type Project struct {
+	BaseCommand
+}
+
+/*
+Here is a sample config file
+
+```
+version: 1.0
+namespace: project-name
+scripts:
+  build: "some build command && another build command"
+  start: "bin/start.sh"
+  stop:  "docker-compose stop"
+  clean: "./cleanup.sh"
+```
+*/
+
+func (cmd *Project) Commands() cli.Command {
+	flags := []cli.Flag{
+		cli.StringFlag{
+			Name:   "config",
+			Value:  "./.outrigger.yml",
+			Usage:  "Path to the project-specific configuration for Outrigger.",
+			EnvVar: "RIG_PROJECT_CONFIG_FILE",
+		},
+	}
+
+	command := cli.Command{
```

Comment:
`rig project run` lists the scripts. `rig project run build` would run the build script.

I wanted to have `rig project create` to run the generator, and possibly other "built-in" commands that are project centric.

There could also be `rig project aliases` and so on. If we are okay with `rig -h` getting longer at the top, we could expand on the "Development" help category and just have create-project, project-aliases, etc as top-level.

I was also making space if we ever wanted to implement lifecycle scripts (maybe via a `hook` or `events` key) to do things like pre-start, post-start around `rig start`. But again, we can implement any commands that might generate on the top level.